src/libs/antares/study/study.cpp

```diff
@@ -1422,13 +1422,15 @@ void Study::computePThetaInfForThermalClusters() const
         // Alias de la zone courant
         const auto& area = *(this->areas.byIndex[i]);
 
-        for (auto& cluster: area.thermal.list.each_enabled_and_not_mustrun())
+        for (auto& c: area.thermal.list.each_enabled_and_not_mustrun())
         {
             for (uint k = 0; k < HOURS_PER_YEAR; k++)
             {
-                cluster->PthetaInf[k] = cluster->modulation[Data::thermalMinGenModulation][k]
-                                        * cluster->unitCount * cluster->nominalCapacity;
+                c->PthetaInf[k] = c->modulation[Data::thermalMinGenModulation][k] * c->unitCount
+                                  * c->nominalCapacity;
             }
+
+            c->anyMinGen(std::ranges::any_of(c->PthetaInf, [](double& e) { return e > 0.5; }));
```

Comment:
This threshold is completely arbitrary

Reply:
Done : replaced with `return !Utils::isZero(e);`